Updates to about and news banner (#20)
NCAR/geocat-hugo: content/about/_index.md

GeoCAT aims to create scalable data analysis and visualization tools for Earth System Science data to serve the geosciences community in the scientific Python ecosystem. GeoCAT tools are built upon cornerstone technologies in the Pangeo stack such as Xarray, Dask, and Jupyter Notebooks. Novel research and development is conducted for analyzing and visualizing structured as well as unstructured grid data from various research fields such as climate, weather, atmosphere, ocean, etc. In addition, some of the functionalities in the GeoCAT stack are inspired/reimplemented from NCL (NCAR Command Language).

The GeoCAT team is committed to open development, prioritizing community involvement at all levels of the project lifecycle, from requirements and specification to testing and deployment, alongside having the whole software stack open-sourced.

> GeoCAT is a team within NSF NCAR's [Visualization & Analysis Systems Technologies (VAST)](https://vast.ucar.edu/). VAST, and in turn GeoCAT, are within the [Computational and Information Systems Lab (CISL)](https://www.cisl.ucar.edu/). as shown in the [UCAR org chart](https://ncar.ucar.edu/who-we-are/org-chart).

Reviewer: ```suggestion
GeoCAT is a team within NSF NCAR's [Visualization & Analysis Systems Technologies (VAST)](https://vast.ucar.edu/) section of the [Computational and Information Systems Lab (CISL)](https://www.cisl.ucar.edu/).
```
